app/javascript/styles/fonts/montserrat.scss

```diff
@@ -0,0 +1,11 @@
+@font-face {
```

Comment:
I guess I would have assumed that the JS moves to app/javascript but anything which really is still just a static asset either a) stays in app/assets (and is made available to the JS, where needed? I think webpack does this...?), or b) moves to like - `app/fonts` or something?

The thing I'm asking about here isn't the move away from sprockets and to webpack, but about the naming convention of having fonts and images in an `app/javascript` dir...